mycroft/messagebus/message.py

```diff
@@ -22,12 +22,30 @@
 
 
 class Message(object):
+    """This class is used to minipulate data to be sent over the websocket
+
+    Attributes:
+        type: type of data sent within the message.
+        data: data sent within the message
+        context: info about the message not part of data such as source,
+            destination or domain.
+    """
     def __init__(self, type, data={}, context=None):
+        """Used to construct a message object
+
+        Message objects will be used to send information back and fourth
+        bettween processes of mycroft service, voice, skill and cli
+        """
         self.type = type
```

Comment:
Not sure if docstring for __init__ is needed since you have a block for the class.

It's missing Args section

between is misspelled